src/main/java/com/epam/edumanagementsystem/student/rest/api/StudentController.java

```diff
@@ -60,25 +67,29 @@ public String openStudentSection(Model model) {
         model.addAttribute(GENDER, Gender.values());
         model.addAttribute(PARENTS, parentService.findAll());
         model.addAttribute(CLASSES, academicClassService.findAll());
+        logger.info("user opened students page . . . ");
         return STUDENT_HTML;
     }
 
     @GetMapping("/without/parent")
     public String openStudentsWithoutParent(Model model) {
         model.addAttribute(STUDENTS, studentService.findStudentsWithoutParent());
+        logger.info("user opened students page without parent . . . ");
+
         return FILTERED_STUDENTS;
     }
 
     @PostMapping
     @Operation(summary = "Creates a new student and saves in DB")
-    public String createStudentSection(@ModelAttribute("student") @Valid StudentDto studentDto,
+    public String saveStudent(@ModelAttribute("student") @Valid StudentDto studentDto,
                                        BindingResult result,
                                        @RequestParam(value = "picture", required = false) MultipartFile multipartFile,
                                        @RequestParam(value = "status", required = false) String status,
                                        Model model) throws IOException {
         if (!multipartFile.isEmpty()) {
             UserDataValidation.validateImage(multipartFile, model);
         }
+        logger.debug(multipartFile.getName()+" "+studentDto.getNameAndSurname());
         if (status.equals("validationFail")) {
             model.addAttribute("size", "File size exceeds maximum 2mb limit");
```

Comment:
Vlidation for multipart file could be taken to a separrate method.Actually, there is one in UserValidation class.